src/api/VoteMonitor.Api.Location/Handlers/PollingStationHandler.cs

```diff
@@ -27,43 +28,17 @@ public PollingStationHandler(VoteMonitorContext context, IMapper mapper, ILogger
 
         public async Task<int> Handle(PollingStationCommand request, CancellationToken cancellationToken)
         {
-            var random = new Random();
 
             try
             {
-                //import the new entities
                 using (var transaction = await _context.Database.BeginTransactionAsync(cancellationToken))
                 {
-                    var id = 100;
-                    var newPollingStations = new List<PollingStation>();
-                    var counties = _context.Counties.ToDictionary(c => c.Code, c => c.Id);
-
-                    foreach (var record in request.PollingStationsDTOs)
-                    {
-                        var pollingStation = _mapper.Map<PollingStation>(record);
-                        pollingStation.Id = id++;
-                        pollingStation.IdCounty = counties[record.CodJudet];//county.Id;
-                        pollingStation.Coordinates = null;
-                        pollingStation.TerritoryCode = random.Next(10000).ToString();
-
-                        newPollingStations.Add(pollingStation);
-                    }
+                    var countiesFromDatabase = _context.Counties.ToList();
```

Comment:
These changes were also in this PR: https://github.com/code4romania/monitorizare-vot/pull/318/files.
Should we remove that and review/merge these changes from this one?
Otherwise, please, create a new branch only with the changes to this feature.

Reply:
Yes, I somehow messed it up, my bad - will fix it - what I did was to merge the changes from the other PR into develop branch from the forked repo, and create a pull request from there - after that, I created a branch from the forked develop, and voila :) 
Will fix it as soon as possible - if meantime that pull request gets closed / merged, then everything will come back to normal. For sure next time I will double check these type of operations. 
Cool that you figured out the relation between pull requests, thanks!